support-frontend/assets/components/directDebit/__tests__/directDebitReducerTest.ts

```diff
@@ -13,16 +14,24 @@ import {
 	updateSortCodeString,
 } from '../directDebitActions';
 import { directDebitReducer as reducer } from '../directDebitReducer';
+
 // ----- Tests ----- //
+
 describe('direct debit reducer tests', () => {
+	const nonExistentAction = {
+		type: 'DOES_NOT_EXIST',
+	} as unknown as Action;
+
 	it('should return the initial state', () => {
-		expect(reducer(undefined, {})).toMatchSnapshot();
+		expect(reducer(undefined, nonExistentAction)).toMatchSnapshot();
 	});
+
```

Comment:
I'm not convinced that this test is that useful, given we're forcing the typesystem to let us use and invalid action so that the switch statement falls back to the default case. For now I've kept it and followed the precedent [here](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/user/__tests__/userReducerTest.ts#L6) for typing the invalid action

Reply:
We can probably get rid of most of these reducer tests in future once we've moved over to Redux Toolkit- at the moment they're right on the edge of testing 'does Redux do its job properly', which IMO is not our responsibility to write unit tests for!